src/main/java/net/boomerangplatform/migration/changesets/flow/FlowDatabaseChangeLog.java

```diff
@@ -1496,16 +1499,35 @@ public void migrationWorkflowScheduleTriggers(MongoDatabase db) throws IOExcepti
           }
 
           schedule.put("timezone", ts.get("timezone"));
-          schedule.put("cronSchedlue", ts.get("schedule"));
+          schedule.put("cronSchedule", ts.get("schedule"));
+
+          schedule.put("workflowId", workflowEntity.get("_id").toString());
+          schedule.put("name", "Migrated Schedule");
+          schedule.put("description", "");
+          schedule.put("creationDate", new Date());
+          schedule.put("labels", new ArrayList<>());
+
+          List<Document> parameters = new ArrayList<>();
+          for (Document property : (List<Document>) workflowEntity.get("properties")) {
+            Document parameter = new Document();
+            parameter.put("key", property.get("key"));
+            parameter.put("value", property.get("defaultValue"));
+            parameters.add(parameter);
+          }
 
-          schedule.put("_id", wfEntity.get("_id"));
+          schedule.put("parameters", parameters);
 
           workflowScheduleCollection.insertOne(schedule);
         }
       }
+      triggers.remove("scheduler");
```

Comment:
Does this leave the scheduler trigger enabled? there is still

```
{
"enable" : true
}
```

Reply:
it will remove the the entire `scheduler` object from `workflow.triggers`

Reply:
Hi @amhudson so we need to keep the scheduler object with enable = true

Reply:
@tlawrie so remove if false?

Reply:
Well we just need to trim the model that's kept under the trigger.

So whilst we migrate the schedule. The trigger still needs to be enabled if it was enabled previously. 

In that the trigger still needs an enable: true|false underneath it in the database

Reply:
@tlawrie 
So just remove `schedule`, `timezone`, and `advancedCron` and leave `enable`?

![image](https://user-images.githubusercontent.com/26099173/150546118-fe86a79f-5961-4ff7-9668-444b3a088dd6.png)